app/models/grda_warehouse/hud/client.rb

```diff
@@ -941,6 +941,7 @@ def scope_for_residential_enrollments
 
     alias_attribute :last_name, :LastName
     alias_attribute :first_name, :FirstName
+    alias_attribute :name_suffix, :NameSuffix
```

Comment:
I don't think this should be required (we should be hitting Hmis::Hud::Client, not the warehouse version.

Reply:
Oh okay, gotcha. Just tested, and it's not needed. removed